.github/workflows/docbuild.yml

```diff
@@ -1,26 +1,31 @@
 name: Documentation Build
 
-on: [push, pull_request, workflow_dispatch]
+on:
+  pull_request:
+  push:
+    tags:
+      - "*"
 
 jobs:
   docs_build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
 
     env:
       PYANSYS_OFF_SCREEN: True
```

Comment:
PyAnsys?

Reply:
Lots of copy pasta. Thanks for catching it.

Reply:
```suggestion
      PYVISTA_OFF_SCREEN: True
```